app/api/v2/statistics.py

```diff
@@ -67,15 +67,16 @@ def on_get(self, req, res, contract_address=None):
             raise DataNotExistsError('contract_address: %s' % contract_address)
 
         # Get dex address
-        TokenContract = Contract.get_contract('IbetStandardTokenInterface', contract_address)
+        TokenContract = Contract.get_contract(
+            'IbetStandardTokenInterface', contract_address)
         dex_address = TokenContract.functions.tradableExchange().call()
 
         # Get holders count
         holders_count = session.query(func.count()). \
             filter(IDXPosition.token_address == contract_address). \
             filter(IDXPosition.account_address != owner_address). \
             filter(IDXPosition.account_address != dex_address). \
```

Comment:
This filter is unnecessary because DEX's position is not stored database.

Reply:
exactly. I'll delete it.

Reply:
deleted